src/rlx_config.erl

```diff
@@ -119,8 +119,14 @@ load({check_for_undefined_functions, CheckForUndefinedFunctions}, {ok, State}) -
     {ok, rlx_state:check_for_undefined_functions(State, CheckForUndefinedFunctions)};
 load({include_erts, IncludeErts}, {ok, State}) ->
     {ok, rlx_state:include_erts(State, IncludeErts)};
-load({system_libs, SystemLibs}, {ok, State}) ->
-    {ok, rlx_state:system_libs(State, SystemLibs)};
+load({system_libs, SystemLibs}, {ok, State}) when is_boolean(SystemLibs) ;
+                                                  is_list(SystemLibs) ->
+    case filelib:is_dir(SystemLibs) of
+        true ->
+            {ok, rlx_state:system_libs(State, SystemLibs)};
+        false ->
+            erlang:error(?RLX_ERROR({bad_system_libs, SystemLibs}))
```

Comment:
Given this, the `is_boolean(SystemLibs)` can be true to enter this clause, but can never actually succeed if it passes. Is that on purpose to show the error?

Reply:
Ah, yup, had before done a list match first in the function header but removed that. Fixing and will add a test to the added shell test that sets it to false to verify this doesn't break again.

Reply:
fixed